packages/invoices/src/InboundShipment/DetailView/AppBarButtons.tsx

```diff
@@ -7,7 +7,7 @@ import {
   useDetailPanel,
   useTranslation,
 } from '@openmsupply-client/common';
-import { useIsInboundEditable } from '../api';
+import { useIsInboundDisabled } from '../api';
```

Comment:
From here down is the same as stocktake.

Add a hook to list views to disable 'disabled' rows.